api/src/main/java/org/openmrs/notification/db/hibernate/HibernateSentMessageDAO.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.notification.db.hibernate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.hibernate.SessionFactory;
+import org.openmrs.User;
+import org.openmrs.api.db.DAOException;
+import org.openmrs.notification.SentMessage;
+import org.openmrs.notification.db.SentMessageDAO;
+
+/**
+ * Hibernate specific implementation of the SentMessageDAO
+ * 
+ * @see org.openmrs.notification.db.SentMessageDAO
+ */
+public class HibernateSentMessageDAO implements SentMessageDAO {
+
+	private final Log log = LogFactory.getLog(getClass());
+	
+	/**
+	 * Hibernate session factory
+	 */
+	private SessionFactory sessionFactory;
+	
+	public HibernateSentMessageDAO() {}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/openmrs/openmrs-core/file/2318749637/issues/source?bid=3204099&fileBranchId=3204129#l37)